manifests/init.pp

```diff
@@ -37,7 +37,7 @@
   require chocolatey
 
   $_chocolatey_server_location      = $server_install_location
-  $_chocolatey_server_app_pool_name = 'chocolatey.server'
+  $_chocolatey_server_app_pool_name = 'chocolateyserver'
```

Comment:
Why the rename?

Reply:
I was getting this error and its a puppetlabs/iis thing.

`Error: Parameter name failed on Iis_application_pool[chocolatey.server]: name is not a valid web site name at C:/puppet/development/modules/chocolatey_server/manifests/init.pp:78`

Reply:
There's some open PRs that lessen the strictness of the name validation. You might not have to rename this later